local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -749,15 +718,12 @@ void LocalPlannerNode::publishGround() {
 }
 
 void LocalPlannerNode::printPointInfo(double x, double y, double z) {
-  geometry_msgs::PoseStamped drone_pos = local_planner_->getPosition();
+  Eigen::Vector3f drone_pos = local_planner_->getPosition();
   int beta_z = floor(
-      (atan2(x - drone_pos.pose.position.x, y - drone_pos.pose.position.y) *
+      (atan2(x - drone_pos.x(), y - drone_pos.y()) *
        180.0 / M_PI));  //(-180. +180]
-  int beta_e = floor((atan((z - drone_pos.pose.position.z) /
-                           sqrt((x - drone_pos.pose.position.x) *
-                                    (x - drone_pos.pose.position.x) +
-                                (y - drone_pos.pose.position.y) *
-                                    (y - drone_pos.pose.position.y))) *
+  int beta_e = floor((atan((z - drone_pos.z()) / sqrt((x - drone_pos.x()) *
```

Comment:
`sqrt((x - drone_pos.x()) * (x - drone_pos.x()) + (y - drone_pos.y()) * (y - drone_pos.y()))`

can be replaced with

`(Eigen::Vector2f(x,y) - drone_pos.topRows<2>()).norm()` 

which I think is a lot more readable